src/FSharpSource.targets

```diff
@@ -185,6 +185,7 @@
     <DefineConstants>$(DefineConstants);NETSTANDARD1_6</DefineConstants>
     <DefineConstants>$(DefineConstants);NETCOREAPP1_0</DefineConstants>
     <OtherFlags>$(OtherFlags) --simpleresolution</OtherFlags>
+    <TargetProfile>netcore</TargetProfile>
```

Comment:
@dsyme thx, i'll do.
 i was asking usage times ago, adding ref to https://github.com/dotnet/netcorecli-fsc/issues/2 for remeber to do it